### PR TITLE
Refine log level of datanode and datacoord

### PR DESCRIPTION
**Pull request by czs007**
Signed-off-by: zhenshan.cao <zhenshan.cao@zilliz.com>

### CODE REVIEW

File: internal/datanode/flow_graph_delete_node.go [hunk 1]
```diff
@@ -122,7 +122,11 @@ func (dn *deleteNode) Operate(in []Msg) []Msg {
 	//then we will add all segments in the fgMsg.segmentsToFlush into the toFlushSeg and remove duplicate segments
 	//the aim for taking all these actions is to guarantee that the memory consumed by delBuf will not exceed a limit
 	segmentsToFlush := dn.delBufferManager.ShouldFlushSegments()
-	log.Info("should flush segments, ", zap.Int("seg_count", len(segmentsToFlush)))
+	if len(segmentsToFlush) == 0 {
+		log.RatedDebug(10.0, "should flush segments, ", zap.Int("seg_count", len(segmentsToFlush)))
```
Comment (XuanYang-cn) [inline internal/datanode/flow_graph_delete_node.go:126]: please don't log empty segmentsToFLush

Comment (czs007) [reply internal/datanode/flow_graph_delete_node.go:126]: OK！

**Pull request closed without merging**